server/service/queries.go

```diff
@@ -177,6 +177,14 @@ func (svc *Service) GetQueryReportResults(ctx context.Context, id uint) ([]fleet
 	return queryReportResults, nil
 }
 
+func (svc *Service) QueryReportIsClipped(ctx context.Context, queryID uint) (bool, error) {
```

Comment:
Missing authorization check:
```go
query, err := svc.ds.Query(ctx, queryID)
if err != nil {
  setAuthCheckedOnPreAuthErr(ctx)
  return nil, nil, ctxerr.Wrap(ctx, err, "get query from datastore")
}
if err := svc.authz.Authorize(ctx, query, fleet.ActionRead); err != nil {
  return nil, nil, err
}
```

(In case this is used in the future for other endpoints.)